pyvrp/Model.py

```diff
@@ -113,7 +113,7 @@ def add_client(
         demand: int = 0,
         service_duration: int = 0,
         tw_early: int = 0,
-        tw_late: int = 0,
```

Comment:
I don't understand why we initialize `tw_late` and others with `INT_MAX` instead of `MAX_USER_VALUE`. More generally, why do we not check and warn about all user data to be smaller than `MAX_USER_VALUE`? Is that only an issue for duration/distances?

Reply:
Don't have strong opinions on this. I think it was mostly meant as a convenience for the user when setting duration/distance data. We might want to build something like this in more generally when constructing a `ProblemData` instance, but that's not very high on my TODO list.

Reply:
I'm also OK with not warning about anything, and removing the warnings we currently have, if we can make things work nicely with potential overflow.